reporting/impl/src/main/resources/reports/resources/css/tech-report-punchcard.css

```diff
@@ -6,7 +6,10 @@
 .sectorStats   { color: black; }
 
 /* A subtle line and space between sectors. */
-.sector           {border-left: none;}
+.sector           {
+    border-left: none;
+    cursor: pointer;
```

Comment:
For the same reason as the previous comment, better add the `cursor: pointer;` to `.headersGroup .sector` selector so that not every bubble has the pointer.